### DISCUSSION TITLE
Merging MTS files for Davinci Resolve doesn't work but a binary copy from the command prompt does.

### DISCUSSION THREAD

**Discussion by DavidOpgh** [mifi/lossless-cut#2219] [Q&A]
I was trying to use Lossess-cut to merge MTS video files from my camcorder for DaVinci Resolve. 

The resulting merged MTS files is not recognized by Resolve. But if I do a binary copy from the command prompt i.e. 
copy /b 00000.mts+00001.mts NewFile.MTS the resulting file is recognized by Resolve.

I tried enabling the following options but it didn't fix the issue

**_Include all tracks If this is checked, all audio/video/subtitle/data tracks will be included. This may not always work for all file types. If not checked, only default streams will be included.

Preserve original metadata when merging (slow)_**

Any thoughts what the issue might be?

**Comment by mifi** [comment]
which output format did you select in losslesscut? `mpegts`? you can try some of the suggestions here: https://github.com/mifi/lossless-cut/blob/master/issues.md#mpeg-ts--mts

if none of it works, then maybe LosslessCut should also have the possibility to simply binary concat MPEGTS files like you did.
